src/components/pages/InstructorAddCourse/InstructorAddCourseCards.js

```diff
@@ -76,124 +66,92 @@ const InstructorAddCourseCards = props => {
           layout="horizontal"
           onChange={handleChange}
         >
-          <Form.Item lable="Select">
-            <label htmlFor="subject">Subject: </label>
+          <Form.Item>
+            <label for="courseType">Course Type: </label>
+            <Select>
+              {/* we need to create an endpoint with an instructor's approved courses for this */}
+            </Select>
+          </Form.Item>
+
+          <Form.Item>
+            <label for="day">Day: </label>
             <Select
               onChange={value => {
-                setState({
-                  ...state,
-                  subject: value,
+                setClassData({
+                  ...classData,
+                  days: value,
```

Comment:
Just noticed that the initial state has this key as "day" rather than the plural form used here. Would we want to spread the current days in the array before adding the new value?

Reply:
Great catch thanks! Fixed in next commit.